include/pgettext.php

```diff
@@ -44,7 +44,7 @@ function get_browser_language() {
 	// check if we have translations for the preferred languages and pick the 1st that has
 	for ($i=0; $i<count($lang_list); $i++) {
 		$lang = $lang_list[$i];
-		if(file_exists("view/lang/$lang") && is_dir("view/lang/$lang")) {
+		if($lang === 'en' || file_exists("view/lang/$lang") && is_dir("view/lang/$lang")) {
```

Comment:
I'm always confused when there are multiple different logical operators and no brackets to clearly show the structure.

I would prefer it that way:
```if ($lang === 'en' || (file_exists("view/lang/$lang") && is_dir("view/lang/$lang")))```

And BTW: I'm expecting that you are talking with yourself about standards? :-)